examples/api_discovery.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Reads the list of the available API versions and prints them.
```

Comment:
Reads the list of available API versions

Reply:
Changed.